app/assets/v2/js/pages/townsquare.js

```diff
@@ -1,5 +1,36 @@
 $(document).ready(function() {
 
+  let resizeTimeout = null;
+
+  const collapseAccordions = function() {
+    const window_width = $('body').width();
+
+    $('.townsquare_block-header').each(function(e) {
+      const target_id = $(this).data('target');
+      const item = localStorage.getItem(target_id.replace(/^#/, ''));
+
+      if (window_width <= 992) {
+        if (item && item !== 'false' && !$(this).hasClass('collapsed')) {
```

Comment:
redundant `if`! 
can be combined with L12

Reply:
This was because we didn't want to ever follow the else path if window_width was less than 992 -- I've tidied up the logic and added comments to explain it here: bce95beb6f0bd358b1cef028a0cbd0ac806d48e8